### PR TITLE
aggregate apr & max earnings

### PR DESCRIPTION
**Pull request by namgold**

### CODE REVIEW

File: src/hooks/useAggregatorAPR.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    is_farm: boolean
+  }
+  total_earnings: number
+}
+
+export default function useAggregatorAPR(): APRResponse {
+  const fetcher = (url: string) => fetch(url).then(r => r.json())
+
+  // const url = `${process.env.REACT_APP_AGGREGATOR_STATS_API}/api/max-apr-and-total-earnings`
+  const url = `https://aggregator-stats.dev.kyberengineering.io/api/max-apr-and-total-earning`
```
Comment (tiendv89) [inline src/hooks/useAggregatorAPR.ts:17]: update the url please

File: src/components/YieldPools/index.tsx [hunk 2]
```diff
@@ -119,8 +121,8 @@ const YieldPools = ({ loading, active }: { loading: boolean; active?: boolean })
   const noFarms = !Object.keys(farms).length
 
   useEffect(() => {
-    setSearchText('')
-  }, [active])
+    setSearchText(search ?? '')
```
Comment (viet-nv) [inline src/components/YieldPools/index.tsx:124]: why do you need setSearchText here? fully use query param please
(push search string to url when user search)

**Review by viet-nv — CHANGES_REQUESTED** [review]

File: src/components/YieldPools/index.tsx (hunk 2 above)
Comment (viet-nv) [reply src/components/YieldPools/index.tsx:124]: https://github.com/KyberNetwork/dmm-interface/blob/add-liquid-pro-amm/src/pages/Pools/index.tsx#L47..L51
An example for you

Comment (namgold) [reply src/components/YieldPools/index.tsx]: Fixed that @viet-nv

**Review by viet-nv — APPROVED** [review]

**Pull request merged**